pkgs/by-name/ha/hauk/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "hauk";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "bilde2910";
+    repo = "Hauk";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-h5OXAFkaQ1nmeF6AjFQ81JCD8mZCIZAPz0JEK9cwghI=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/
+    cp -R ./backend-php/* $out/
+    cp -R ./frontend/* $out/
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Hauk is a fully open source, self-hosted location sharing service.";
```

```suggestion
    description = "Hauk is a fully open source, self-hosted location sharing service";
```
See https://nixos.org/manual/nixpkgs/stable/#var-meta-description